sumologic/provider.go

```diff
@@ -134,22 +143,37 @@ func resolveRedirectURL(accessId string, accessKey string) (string, error) {
 func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 	accessId := d.Get("access_id").(string)
 	accessKey := d.Get("access_key").(string)
+	authJwt := d.Get("auth_jwt").(string)
 	environment := d.Get("environment").(string)
 	baseUrl := d.Get("base_url").(string)
 	isInAdminMode := d.Get("admin_mode").(bool)
 
 	msg := ""
-	if accessId == "" {
-		msg = "sumologic provider: access_id should be set;"
-	}
-
-	if accessKey == "" {
-		msg = fmt.Sprintf("%s access_key should be set; ", msg)
+	if authJwt == "" {
+		if accessId == "" || accessKey == "" {
+			msg = "sumologic provider: auth_jwt is not set;"
```

Comment:
Since we are not documenting JWT authentication at this point and only using it internally, if both auth JWT and access id/key are not specified, we probably do not want to display `auth_jwt is not set;` in the error message. The most likely scenerio is that a user forgot to specify the access id/key, and the proper response is that the access id / key is missing. I.e. IMO we should not expose `auth_jwt` in any public response unless 100% sure that the user intended to use JWT auth.

Reply:
+1. We should not expose JWT auth  in error messages.

Reply:
Done.